Bindings/bindings.json

```diff
@@ -383,6 +383,9 @@
                         "capability": "blob"
                     }
                 }
+            ],
+            "filters": [
+                "showHiddenBinding"                
```

Comment:
In your email you said that you're hiding it from prod and staging, but this filter says "showHiddenBinding".  Doesn't that contradict what you said?  Regardless, we actually want to expose the API Hub File bindings so that we can start getting feedback on them.